### PR TITLE
FHIR Diagnostic Resource, Profile, Mapping, Example resource

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/FM2-502  cc @mozzy11 , @ibacher  @pmanko 
diagnosticResort

### CODE REVIEW

File: input/fsh/omrs-diagnosticReport.fsh [hunk 1]
```diff
@@ -3,6 +3,58 @@ Parent: DiagnosticReport
...
+* media 0..*
+* conclusion 0..1
+* presentedForm 0..*
+
+Instance: example-Openmrs-DiagnosticReport
+InstanceOf: OpenMRSDiagnosticReport
+Usage: #example
+Title: "Openmrs DiagnosticReport"
+Description: "Example OMRS DiagnosticReport Resource"
+* identifier.id = "53fbd0ef-ae10-46a0-9d0a-387917a66d6e"
```
Comment (pmanko) [inline input/fsh/omrs-diagnosticReport.fsh:36]: Does the FHIR2 module translate into a diagnostic report with an identifier? I don't see it anywhere in the translator: https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/DiagnosticReportTranslatorImpl.java

Comment (pmanko) [reply input/fsh/omrs-diagnosticReport.fsh:36]: I also don't see any identifiers in the example DR from the fhir2 tests: https://github.com/openmrs/openmrs-module-fhir2/blob/master/test-data/src/test/resources/org/openmrs/module/fhir2/providers/DiagnosticReport_create_r4.json

**Review by pmanko — CHANGES_REQUESTED** [review]
Looks good, but I'm not sure about the identifier for the DiagnosticReport. Other fields also don't seem to be supported in the fhir2 module - for example `resultsInterpreter` and `imagingStudy`. In this IG we should only list the fields that our module supports.

**Comment by sherrif10** [comment]
cc @pmanko . Looking forward for your reviews.

**Pull request merged**